cypress/e2e/orange/file.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Candidate Interview uploade file
```

Comment:
one general note, you should have the sentences more efficient, like if the system should have some data you should say `The system has a ... record` something like this,